lib/takeaway.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'menu'
+require 'twilio-ruby'
+
+class Takeaway
+
+  ACCOUNTS_SID = ENV['TWILIO_SID']
+  AUTH_TOKEN = ENV['TWILIO_AUTH']
+
+  attr_reader :menu
+
+  def initialize(menu_klass = Menu.new, twilio_klass = Twilio::REST::Client.new(ACCOUNTS_SID, AUTH_TOKEN))
+    @menu = menu_klass
+    @twilio = twilio_klass
+  end
+
+  def add(food, quantity = 1)
+    @menu.add(food, quantity)
+  end
+
+  def request_menu
+    @menu.view_menu
+  end
+
+  def view_current_order
+    @menu.view_basket
+  end
+
+  def checkout(total)
+    raise "Total provided does not match order total!  Please try again" unless confirm_total(total)
```

Comment:
Metrics/LineLength: Line is too long. [100/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)